README.md

```diff
@@ -81,20 +124,27 @@ Feel free to check the [issues page](../../issues/).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+
 ## ⭐️ Show your support <a name="support"></a>
 
-If you like this project please let me know with a recomendation, i will be proud.
+
+If you like this project let me know with a STAR!
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-I would like to thank all my coding partners who were there to help and support me.
+
+I would like to thank all my coding partners for giving to me your help and support.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
 
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```

Comment:
- [ ] [OPTIONAL] Kindly, remove this statement as this is just a description and shouldn't be in your Professional "Readme" 😉